packages/components/src/FormField/FormField.css

```diff
@@ -326,3 +339,21 @@
 .description {
   margin-top: var(--space-smaller);
 }
+
+.toolbar {
+  display: flex;
+  position: relative;
+  margin: var(--space-small) var(--space-base) var(--space-base) var(--space-base);
+}
+
+.toolbar::before {
+  content: "";
+  display: block;
+  position: absolute;
+  top: calc(-1.25 * var(--space-base));
```

Comment:
this 1.25 is a bit magical

I don't even remember why I chose that value anymore. could have simply been this is when it started showing up